addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -256,9 +256,9 @@ def _prepare_down_payment_lines_values(self, order):
         computed_taxes = self.env['account.tax']._compute_taxes(tax_base_line_dicts, self.company_id)
         down_payment_values = []
         for line, tax_repartition in computed_taxes['base_lines_to_update']:
-            account = line['product'].product_tmpl_id.get_product_accounts(
-                fiscal_pos=order.fiscal_position_id
-            ).get('income')
+            product_account = line['product'].product_tmpl_id.get_product_accounts(fiscal_pos=order.fiscal_position_id)
+            account = product_account.get('downpayment') or product_account.get('income')
```

Comment:
how can this work (I know it was like this before) when the code is calling `get_product_accounts` (without `_`) but the method defined on product.template (see sale/modelsproduct_template.py) has it???

very very louche

Reply:
It's because it will call the method here in account/models/products.py which is in the product.template model
![image](https://github.com/user-attachments/assets/ddd1176b-f314-4af8-88b2-95732ce71ac5)

I can do something about it if you want ahah